### DISCUSSION TITLE
multiple different fonts

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [mantinedev/mantine#1419] [Q&A] [answered]
Hello. I have a css file that contains multiple different fonts configuration. How to use these multiple fonts in mantine?

**Comment by rtivital** [comment, marked as answer]
You can add font-family styles to any component with createStyles/sx

`<Button sx={{ fontFamily: 'Roboto' }} />`

**Reply by MadadiSaeed** [reply]
I don't know why, but It doesn't apply on TextInput component placholder's font or value's font. I am using sx directly in TextInput iteself. It is working on Button component.

**Update**: I am sorry.  Now I know how to use sx. But how to apply font family dynamically based on text direction globally?

**Reply by rtivital** [reply]
`<Button sx={{ fontFamily: condition ? 'Roboto' : 'Raleway' }} />`
